libfreerdp/codec/h264.c

```diff
@@ -595,6 +595,13 @@ static BOOL CALLBACK h264_register_subsystems(PINIT_ONCE once, PVOID param, PVOI
 		subSystems[i] = &g_Subsystem_libavcodec;
 		i++;
 	}
+#endif
+#ifdef WITH_MEDIACODEC
+	{
+		extern H264_CONTEXT_SUBSYSTEM g_Subsystem_mediacodec;
```

Comment:
Order is important here. Sure you only want `MEDIACODEC` as fallback? I´d move it to the top, e.g. if it is available it is the preferred method, otherwise fall back to `openh264` or `ffmpeg`